EdFi.Ods.Utilities.Migration/Scripts/PgSql/02Upgrade/v51_to_v52/01 Structure/1010-AddNewTablesForV52.sql

```diff
@@ -0,0 +1,245 @@
+    CreateDate                      TIMESTAMP    DEFAULT current_timestamp NOT NULL,
+    CONSTRAINT StaffAncestryEthnicOrigin_PK PRIMARY KEY  (AncestryEthnicOriginDescriptorId , StaffUSI ),
+    CONSTRAINT fk_a4a6ae_ancestryethnicorigindescriptor FOREIGN KEY (AncestryEthnicOriginDescriptorId) REFERENCES edfi.AncestryEthnicOriginDescriptor (AncestryEthnicOriginDescriptorId),
+    CONSTRAINT fk_a4a6ae_staff FOREIGN KEY (StaffUSI) REFERENCES edfi.Staff (StaffUSI) ON DELETE CASCADE
+);
+
+
+CREATE  INDEX fk_a4a6ae_ancestryethnicorigindescriptor
+    ON edfi.StaffAncestryEthnicOrigin(AncestryEthnicOriginDescriptorId );
+
+
+CREATE  INDEX fk_a4a6ae_staff
+    ON edfi.StaffAncestryEthnicOrigin(StaffUSI );
+
+
+CREATE TABLE edfi.StudentDisciplineIncidentBehaviorAssociationDisciplineIn_ae6a21 (
+    BehaviorDescriptorId                            INT           NOT NULL,
+    DisciplineIncidentParticipationCodeDescriptorId INT           NOT NULL,
+    IncidentIdentifier                              VARCHAR (20) NOT NULL,
+    SchoolId                                        INT           NOT NULL,
+    StudentUSI                                      INT           NOT NULL,
+    CreateDate                                      TIMESTAMP    DEFAULT current_timestamp NOT NULL,
+    CONSTRAINT StudentDisciplineIncidentBehaviorAssociationDiscip_ae6a21_PK PRIMARY KEY  (BehaviorDescriptorId , DisciplineIncidentParticipationCodeDescriptorId , IncidentIdentifier , SchoolId , StudentUSI ),
+    CONSTRAINT FK_ae6a21_DisciplineIncidentParticipationCodeDescriptor FOREIGN KEY (DisciplineIncidentParticipationCodeDescriptorId) REFERENCES edfi.DisciplineIncidentParticipationCodeDescriptor (DisciplineIncidentParticipationCodeDescriptorId),
+    CONSTRAINT FK_ae6a21_StudentDisciplineIncidentBehaviorAssociation FOREIGN KEY (BehaviorDescriptorId, IncidentIdentifier, SchoolId, StudentUSI) REFERENCES edfi.StudentDisciplineIncidentBehaviorAssociation (BehaviorDescriptorId, IncidentIdentifier, SchoolId, StudentUSI) ON DELETE CASCADE
+);
+
+
+CREATE  INDEX FK_ae6a21_DisciplineIncidentParticipationCodeDescriptor
+    ON edfi.StudentDisciplineIncidentBehaviorAssociationDisciplineIn_ae6a21(DisciplineIncidentParticipationCodeDescriptorId );
+
+
+CREATE  INDEX FK_ae6a21_StudentDisciplineIncidentBehaviorAssociation
+    ON edfi.StudentDisciplineIncidentBehaviorAssociationDisciplineIn_ae6a21(BehaviorDescriptorId , IncidentIdentifier , SchoolId , StudentUSI );
+
+
+CREATE TABLE edfi.StudentDisciplineIncidentNonOffenderAssociation (
+    IncidentIdentifier VARCHAR (20)    NOT NULL,
+    SchoolId           INT              NOT NULL,
+    StudentUSI         INT              NOT NULL,
+    Discriminator      VARCHAR (128)   NULL,
+    CreateDate         TIMESTAMP    DEFAULT current_timestamp NOT NULL,
+    LastModifiedDate   TIMESTAMP    DEFAULT current_timestamp NOT NULL,
+    Id                 UUID  DEFAULT gen_random_uuid() NOT NULL,
```

Comment:
In ODS for v5.2 from the package, this line seems like it should end as `UUID  DEFAULT public.gen_random_uuid() NOT NULL,`

Reply:
fixed it